### PR TITLE
Shim AudioTrack type removed in TS3.9

### PR DESCRIPTION
**Pull request by sandersn**
AudioTrack isn't standard (only supported in IE and Safari), and is removed from the DOM lib in TS 3.9: microsoft/TSJS-lib-generator#802

This PR puts a copy of the type called VideojsAudioTrack in instead. That should be compatible with the DOM version for older Typescripts, since it's copied directly from the old source.

### CODE REVIEW

**Comment by sandersn** [comment]
DT is fine--not sure what Travis is doing ...

**Comment by joeflateau** [comment]
Is this a test to see if maintainers will approve a PR even if Travis fails? ha

**Review by sroucheray — APPROVED** [review]

**Comment by sroucheray** [comment]
If that's the case, obviously... it failed :joy:

**Pull request merged**
